shared/common-adapters/multi-avatar.js

```diff
@@ -50,6 +50,3 @@ const rightAvatar = {
 }
 
 export default MultiAvatar
```

Comment:
Affected by this eslint import issue: https://github.com/benmosher/eslint-plugin-import/issues/660